internal/builders/container/README.md

```diff
@@ -216,10 +216,10 @@ Inputs:
 
 Secrets:
 
-| Name                | Description                                                                                                            |
-| ------------------- | ---------------------------------------------------------------------------------------------------------------------- |
-| `registry-username` | Username to log in the container registry. Either `registry-username` input or `registry-username` secret is required. |
-| `registry-password` | **(Required)** Password to log in the container registry.                                                              |
+| Name                | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| ------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `registry-username` | Username to log in the container registry. Either `registry-username` input or `registry-username` secret is required. This should only be used for high entropy values such as AWS Access Key as described [here](https://github.com/docker/login-action#aws-elastic-container-registry-ecr). Normal username values could match other input values and cause them to be filtered by GitHub Actions. In those cases, use the `registry-username` input instead. |
```

Comment:
`cause them to be filtered by GitHub Actions` we could even tell upfront it will break the build and it may fail. `filtered by GitHub` may seem feels like a good thing to users - since it's filtering secrets.

Reply:
I updated the wording to make it known your build might fail.

I'll leave it out of known issues for now. I tested it and it seems that GitHub doesn't actually ignore inputs for reusable workflows, only individual job inputs/outputs. I imagine this might change in the future though.